pdf_build_src/process_markdowns.py

```diff
@@ -447,14 +447,11 @@ def process_macros(duplicated_src_dir_path):
             matches = re.findall(re.compile("({{.*?}})", re.DOTALL), contents)
             for m in matches:
                 # Remove macro delimiters to get *just* the function call
-                function_string = m.strip("{} ")
-                # Replace prefix with module name
-                function_string = function_string.replace(
-                    "MACROS___",
-                    "macros."
-                )
-                # Run the function to get the output
-                new = eval(function_string)
+                function_name = m.strip("{} ")
+                # Remove prefix
+                function_name = function_name[len("MACROS___"):]

```

Comment:
I would prefer to avoid changing the logic of the program here, and do 

```suggestion
                function_name = re.sub("^MACROS___", "", function_name)
```
so whatever assumptions brought us to this point, would not truncate function_name "blindly"

Reply:
Done, actually re-used `replace("MACROS__", "")` to be as close as possible to the initial logic.